linux_os/guide/services/obsolete/tftp/tftp_uses_secure_mode_systemd/rule.yml

```diff
@@ -0,0 +1,38 @@
+documentation_complete: true
+
+title: 'Ensure tftp Daemon Uses Secure Mode Using systemd'
```

Comment:
The title is weird: uses ... using

Configure tftp systemd service to use secure mode